Dockerfile

```diff
@@ -1,3 +1,4 @@
 ARG VERSION=2019.05.08
 FROM getsentry/jekyll-base:builder-${VERSION} AS builder
 FROM getsentry/jekyll-base:runtime-${VERSION} AS runtime
+CMD ["/bin/sh", "-c", "sed -i \"s/\\([[:space:]]\\{1,\\}\\)listen 80;/\\1listen ${PORT:-80};/\" /etc/nginx/conf.d/default.conf && exec nginx -g 'daemon off;'"]
```

Comment:
We should just bake this behavior into the `jekyll-base:runtime` image in the entrypoint. This is fine as is, but we can do better. The container should also include `envsubst(1)` which should make this pattern a bit easier.

Reply:
I'll submit that as a fix up. Cannot use `envsubst(1)` as it doesn't support fallback values.

Reply:
You shouldn't need it. :) I can show you what I'm talking about ifyou want. You'd do the fallback logic in the actual entrypoint. No need for it in the template.

Reply:
Oh, right. Yeah, not sure why I didn't go with that as I tried that earlier.

Reply:
:) Yeah, just do the variable stuff inside the bash script before calling envsubst.